src/frontend/Dev.ml

```diff
@@ -9,7 +9,12 @@ type twin = [`Only | `TwinL | `TwinR]
 
 type 'a decl =
   | Hole of [`Rigid | `Flex]
-  | Defn of ResEnv.visibility * [`Transparent | `Opaque] * 'a
+  | Macro of 'a
```

Comment:
Based on our chat, I see what this does and it looks like a good idea! Maybe another name would help, like `Auxiliary`.